console/billing/how-to/change-payment-method.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: How to change your default payment method
+  description: This page explains how to change your default payment method
+content:
+  h1: How to change your payment method
+  paragraph: This page explains how to change your default payment method
+tags: billing change default payment 
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+If you have registered both a **credit card** and a **SEPA mandate**, you can choose the default payment method you wish to use for your Scaleway account.
+
+<Message type="requirement">
```

Comment:
Add here that a valid credit card is always required on the account, even if a sepa is added and setup as the default payment method.